backend-server/src/main/java/org/dice/ida/util/DbUtils.java

```diff
@@ -0,0 +1,7 @@
+package org.dice.ida.util;
+
+public class DbUtils {
+	public static String manageNullValues(String value) {
+		return value.equals("?") ? "<ida-unk>" : value;
```

Comment:
you should handle a range of values here. Including blank spaces, "?" etc. Also, please put these literals in the constants file.